site/api/prisma/schema.prisma

```diff
@@ -116,10 +116,25 @@ model Alias {
   @@map("aliases")
 }
 
+model PersonNationality {
+  id        Int    @id @default(autoincrement())
+  country   String
+  personId  Int @map("person_id")
+  person    Person @relation(fields: [personId], references: [id], onDelete: Cascade)
+  addedById Int?   @map("added_by_id")
+  addedBy   User?  @relation(fields: [addedById], references: [id], onDelete: SetNull)
+
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @default(now()) @updatedAt @map("updated_at")
+  @@index([personId])
+  @@map("person_nationalities")
+}
+
```

Comment:
see above.. could be just a part of the Idol..

and if its about us, the authors, we dont really care about it, and dont need to ssafe it :D